### ISSUE TITLE
'publicly_accessible' argument for 'aws_rds_cluster_instance' resource doesn't not work while modify argument

### ISSUE THREAD

**Issue by ankurpshah** [terraform-providers/terraform-provider-aws#2381]
### Terraform Version
```
terraform -v
Terraform v0.11.0
+ provider.aws v1.3.0
```

### Affected Resource(s)
- aws_rds_cluster_instance

### Terraform Configuration Files
```hcl
resource "aws_rds_cluster_instance" "test" {
.....
  instance_class      = "db.t2.small"
  publicly_accessible = true
  apply_immediately   = true
.....
.....
}
```

### Expected Behavior
Once we create AWS RDS Cluster instance with `publicly_accesible` it should accessible publicly. If we modify `publicly_accesible` to `false` then it should not accessible outside VPC.

### Actual Behavior
After modification of `publicly_accesible` argument from `true` to `false`. It still shows AWS RDS Cluster instance setting to "Publicly Accessible" "Yes". 

### Steps to Reproduce
1. `terraform apply` with `publicly_accesible` argument with `true`.
2. Modify `publicly_accesible` argument value to `false`.
3. Run `terraform apply`.

### References
From [code](https://github.com/terraform-providers/terraform-provider-aws/blob/master/aws/resource_aws_rds_cluster_instance.go#L355-L446), it seems AWS RDS Cluster instance modification ignores `publicly_accesible` flag.

**Comment by bflad** [comment]
The fix for this has been merged and will release with version 1.38.0 of the AWS provider, likely later today.

**Comment by bflad** [comment]
This has been released in [version 1.38.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1380-september-26-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
